acquia/acquia_cms
# ACMS-1030: Update search api, search api solr and search api autocomplete module to latest

**Motivation**
Fixes #[1030](https://backlog.acquia.com/browse/ACMS-1030)

**Proposed changes**
Update Search API, Search API Solr and Search API Autocomplete module to latest.

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch
- Visit module page and verify search api and search api solr module are update.
- Enable starter module all page.

**Merge requirements**
- [Enhancement] Major change, Minor change, Bug, Enhancement, and/or Chore label applied
